pkg/config/protocol/instance.go

```diff
@@ -88,6 +88,8 @@ func Parse(s string) Instance {
 		return Redis
 	case "mysql":
 		return MySQL
+	case "hbone":
```

Comment:
Not sure if this will lead to some undefined behavior, as HBONE is used internally and `Parse` protocol is widely used in different places. Is it better to just handle this in the `describe.go` code?